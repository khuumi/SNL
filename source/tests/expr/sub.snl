12.9 - 987