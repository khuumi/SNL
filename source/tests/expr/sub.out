Binop (Float 12.9) Sub (Int 987)